Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Build Front End Application
+FROM node:12 AS build
+
+RUN mkdir /app 
+
+#USER node
+
+WORKDIR /app/
+
+COPY . .
+
+RUN cd frontend && npm install --quiet --production
+ENV ENV=prod
+ENV ASSET_URL=/
+RUN cd frontend && npm run bundledjango
+
+
+# Build Django Application
+FROM python:3.8-slim as application
+
+# Install python dependencies
+WORKDIR /app
+COPY api/requirements.txt /app/
+RUN apt-get update && apt-get install -y git && pip install -r requirements.txt --no-cache-dir --compile
+
+# Copy the Django app into the imag
+COPY api /app/
```

Comment:
Haven't we already done this above with `COPY . .` or is that just in the build image? If the latter, then we shouldn't copy everything in that step right? We should just copy the frontend dir.

Reply:
Not sure why this got resolved without (seemingly) any change or comment to explain?

Reply:
Are you looking at an old version? There are comments in the dockerfile now? We need to copy the built assets from 1st stage to 2nd

Reply:
Nope, don't think so. The comments don't really explain / answer my question do they?